### ISSUE TITLE
Node stuck to block 700720 000000000000000029d80d3a20baa597c67174545dcdcefe391fd3b6c6705165

### ISSUE THREAD

**Issue by Polve** [bitcoin-sv/bitcoin-sv#219]
My node, running version 1.0.8 is stuck to block hash 000000000000000029d80d3a20baa597c67174545dcdcefe391fd3b6c6705165 that I can't find on any block explorer.

I suppose there has been a fork that the client for some reason was unable to resolve.

I tried to manually invalidate some blocks and reconsider others, but without luck: can you give me any hints on how to go to main chain without syncing from the start?

**Comment by Polve** [comment]
Since no solution was give I had to do a full resync

**Comment by AustEcon** [comment]
This happened to me too. I found the common branch point which was:
`00000000000000000cc69c0629ed03f938656f20243c772c6f18cead28e21769`

And block 700597 (`0000000000000000052c4236c4c34dc7686f8285e2646a584785b8d3b1eb8779`) orphaned this block:
`000000000000000004b3b2429955cd7943137cf4b22aaf139bae0c0a97ef11d3` (height 700597)

But my node was stuck on the wrong chain including: `000000000000000004b3b2429955cd7943137cf4b22aaf139bae0c0a97ef11d3` going up to 700720 where it then stayed indefinitely until I intervened.

On whatsonchain https://whatsonchain.com/block-height/700597 has a pop-up saying it was the 
"First Bitcoin block to earn its processor a higher return in transaction fees"

So looks like the reorg that happened there has caused this issue somehow.

Maybe because this alternate chain runs on for 123 blocks, it exceeds some depth limit that the node tracks to recognize it and recover? 🤷 I don't know. Just a wild ass guess.
